### PR TITLE
imports react-modal as a dependency and sets up dummy modal view for testing

### PR DESCRIPTION
**Pull request by markrivasrobertson**
imported react modal and added to package.json, commented out App component in Index.jsx simply for testing of ModalView (which I imagine we can just comment out and keep a long string of commented versions until we're ready to start putting things together?).  Modal is functional.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ npm-debug.log
 /build
 .eslintrc.js
 .eslintrc.json
+.eslintrc
```
Comment (icsvortex666) [inline .gitignore]: I got a nice config on my .eslintrc (mine is .eslintrc.**js**) file if you want to include it in yours (to hide some annoyances):
```js
module.exports = {
  "extends": "airbnb",
  "installedESLint": true,
  "plugins": [
    "react",
    "jsx-a11y",
    "import",
  ],
  "env": {
    "browser": true,
  },
  "rules": {
    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
  },
  "arrow-body-style": ["error", "never"],
};
```

File: client/components/Modal.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Modal from 'react-modal';
+
+const customStyling = {
```
Comment (icsvortex666) [inline client/components/Modal.jsx]: Do you think we'll be able to move this styles into an external css file?

File: client/components/Modal.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+};
+
+class ModalView extends React.Component {
+  constructor() {
+    super();
+    this.state = { open: false };
+    this.openModal = this.openModal.bind(this);
+    this.closeModal = this.closeModal.bind(this);
+  }
+  openModal() {
```
Comment (icsvortex666) [inline client/components/Modal.jsx]: How about a new line between each function?
e.g:
```js
openModal() {
  …
}

closeModal() {
  …
}
```

File: client/components/Modal.jsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    WebkitOverflowScrolling: 'touch',
+    borderRadius: '4px',
+    outline: 'none',
+    padding: '1rem',
+  },
+};
+
+class ModalView extends React.Component {
+  constructor() {
+    super();
```
Comment (icsvortex666) [inline client/components/Modal.jsx]: How about a new line between super(); and then states. It's nice to separate things that are not related to each other.

File: client/index.jsx [hunk 5]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './components/App.jsx';
+// import App from './components/App.jsx';
+import ModalView from './components/Modal.jsx';
 
-ReactDOM.render(<App />, document.querySelector('#root'));
+// ReactDOM.render(<App />, document.querySelector('#root'));
+ReactDOM.render(<ModalView />, document.querySelector('#root'));
```
Comment (icsvortex666) [inline client/index.jsx]: Why are you switching from App to ModalView?

Just for testing?

File: package.json [hunk 6]
```diff
@@ -7,7 +7,8 @@
   "description": "General Assembly's Student Project Showcase",
```
Comment (icsvortex666) [inline package.json]: Thanks for updating the dependencies!

File: .gitignore (hunk 1 above)
Comment (markrivasrobertson) [inline .gitignore]: thanks, I'll add that when i get back to coding

File: client/components/Modal.jsx (hunk 2 above)
Comment (markrivasrobertson) [inline client/components/Modal.jsx]: I don't know.  Still sort of shaky on how the modal is working, since a lot of it seems to be behind the scenes in the node module.  Maybe I'll spend a little time looking at the source code tomorrow to see if I can make sense of it.  Maybe we can, but the docs sounded like you had to have some styling added though an object literal like that.  I'll take a look.

File: client/components/Modal.jsx (hunk 4 above)
Comment (markrivasrobertson) [inline client/components/Modal.jsx]: fair point

File: client/components/Modal.jsx (hunk 3 above)
Comment (markrivasrobertson) [inline client/components/Modal.jsx]: yep

File: client/index.jsx (hunk 5 above)
Comment (markrivasrobertson) [inline client/index.jsx]: Yes, needed it appended to the DOM to test.  Can be commented back out and then eventually removed when we have other links that will call the modal open

File: client/components/Modal.jsx (hunk 2 above)
Comment (icsvortex666) [inline client/components/Modal.jsx]: Can you send me the docs to get a glimpse on what's going on?
I agree with you though!

File: package.json (hunk 6 above)
Comment (markrivasrobertson) [inline package.json]: no sweat.  just takin' care of business

File: client/index.jsx (hunk 5 above)
Comment (icsvortex666) [inline client/index.jsx]: Fair enough!

File: package.json (hunk 6 above)
Comment (icsvortex666) [inline package.json]: Awesome bro!

**Comment by icsvortex666** [comment]
I went ahead and added +1 and +2. Will merge!

**Comment by markrivasrobertson** [comment]
ok, should I just add those changes you recommended in a different quick branch just for those changes?

**Comment by icsvortex666** [comment]
Yes, go ahead.

**Comment by icsvortex666** [comment]
Let's get our habit of commiting and PR's. It's true you could've just updated that file and then merge but it's fine.
